client/src/components/landing/index.js

```diff
@@ -45,8 +45,8 @@ export const Landing = ({ edges }) => {
                 Earn certifications.
               </h2>
               <h2 className='medium-heading text-center'>
-                Since 2014, more than 40,000 freeCodeCamp.org graduates have
-                gotten jobs at tech companies including:
+                Since 2015, 40,000 graduates have gotten jobs at tech companies
+                including:
```

Comment:
I disagree with these changes. I think the original copy is stronger and more clear. Also, we started in 2014 and being older is more beneficial in this case.

Reply:
Then the copy on news is incorrect? A quick google search for freeCodeCamp give different results.